spec/spec_helper.rb

```diff
@@ -2,6 +2,28 @@
 require 'mumukit/bridge'
 require 'simplecov'
 
+def valid_response(test_name)
```

Comment:
TIP: If it is going to be used only in integration test, add this methods directly to it and not to the spec helper.